tests/provision/beaker/main.fmf

```diff
@@ -1,13 +1,14 @@
 # These tests are manual only for now as they need a running
 # Beaker instance
-enabled: false
+enabled: true
```

Comment:
Now the comment is outdated, correct? It no longer applies, and probably should have been moved together with the `enabled: false` that now lives in `/reconnect`.

Reply:
I don't understand. You removed `enabled: false` which makes all tests enabled - what did it change for the comment? It still says tests are manual only, which is no longer true, all those tests are now enabled.